### ISSUE TITLE
DSSC rectangular pixel problems

### ISSUE THREAD

**Issue by FilipeMaia** [European-XFEL/geoAssembler#34]
When using the notebook interface to geoAssembler with the DSSC detector it seems that `Calibrate` draws the assembled image with `position_modules_fast` which returns an image with rectangular pixels. 
But the circles drawn by Calibrate assume that the image has square pixels, which makes them useless (or even deceiving) for helping with the assembly.

**Comment by takluyver** [comment]
We should make this automatic based on the detector type, but you can pass in `Calibrate(..., aspect=1.5/np.sqrt(3))` to set the aspect ratio (1.5 / sqrt(3) coming from hexagon packing). It looks like this is connected up to the circle drawing, so I think once you pass that, you should be able to use circles as expected.

Can you point me to some data I can look at to try this myself? This is part of proposal 2942, right? If you can give me run, train & pulse numbers I can take a look.

**Comment by FilipeMaia** [comment]
Yes, it's part of the proposal 2942. You could look at run 35, index 11062 for a spherical hit.

**Comment by takluyver** [comment]
PR #35 should set the aspect ratio automatically for DSSC. It looks something like this afterwards (with my guesswork at quadrant positions):

![image](https://user-images.githubusercontent.com/327925/128715480-15e959d0-867d-4488-81d4-803377f207f0.png)
